StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebView.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  ConnectWebView.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 5/3/24.
+//
+
+@_spi(STP) import StripeCore
+import SafariServices
+import WebKit
+
+/**
+ Custom implementation of a web view that handles:
+ - Camera access
+ - Popup windows
+ - Opening email links
+ - Downloads  TODO MXMOBILE-2485
+ */
+class ConnectWebView: WKWebView {
+
+    /// Closure to present a popup web view controller.
+    /// This is required for any components that can open a popup, otherwise an assertionFailure will occur.
+    var presentPopup: ((UIViewController) -> Void)?
+
+    /// Closure that executes when the view finishes loading.
+    /// - Note: If any JS needs to be evaluated immediately after instantiation, do that here.
+    var didFinishLoading: ((ConnectWebView) -> Void)?
+
+    /// Closure that executes when `window.close()` is called in JS
+    var didClose: ((ConnectWebView) -> Void)?
+
+    /// The instance that will handle opening external urls
+    let urlOpener: URLOpener
+    
+    /// The current version for the SDK
+    let bundleVersion: String?
+    
+    init(frame: CGRect,
+         configuration: WKWebViewConfiguration,
+         // Only override for tests
+         urlOpener: URLOpener = UIApplication.shared,
+         bundleVersion: String? = StripeConnectBundleLocator.bundleVersion) {
```

Comment:
🚫 Blocker

We have a helper for this: `StripeAPIConfiguration.STPSDKVersion`. 

We use a hardcoded string instead of the `CFBundleVersion` because there are some cases where the CFBundleVersion doesn't map to the SDK package version, although I don't recall exactly what they were (building from source or maybe certain configurations of Cocoapods / Carthage?).

You'll need to add `@_spi(STP) import StripeCore` to access this:

```suggestion
         sdkVersion: String? = StripeAPIConfiguration.STPSDKVersion) {
```